src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java

```diff
@@ -228,10 +222,8 @@ private void changeNamespaceSetting(RandomDataGenerator random, State state, Ran
       env.getAccumuloClient().namespaceOperations().setProperty(namespace,
           setting.property.getKey(), "" + newValue);
     } catch (AccumuloException ex) {
-      if (ex.getCause() instanceof ThriftTableOperationException) {
-        ThriftTableOperationException ttoe = (ThriftTableOperationException) ex.getCause();
-        if (ttoe.type == TableOperationExceptionType.NAMESPACE_NOTFOUND)
-          return;
+      if (ex.getCause() instanceof TableNotFoundException) {
```

Comment:
This one also seems to be looking for a NamespaceNotFoundException

Reply:
Yup, nice catch.